apps/api/e2e/setup.ts

```diff
@@ -41,34 +41,26 @@ before(async () => {
 
 after(async () => {
   await testServer.teardown();
-  await dropDatabase();
```

Comment:
why dont we want to drop the db after run?

Reply:
In CI, it doesn't matter as the runner is stopped. Locally it helps debugging failed test by checking their status in DB.